pkg/uefivars/boot/boot.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2015-2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+package boot
```

Comment:
can you provide a short package comment? thanks

Reply:
There is a package comment in `pkg/uefivars/boot/doc.go`, though if preferred I can move that to this file and delete doc.go.

Reply:
woops, i just missed it. sorry.